bpfd-operator/test/integration/tc_test.go

```diff
@@ -18,8 +18,8 @@ import (
 )
 
 const (
-	tcCounterBytecodeYaml      = "../../../examples/go-tc-counter/kubernetes-deployment/go-tc-counter-bytecode.yaml"
-	tcGoCounterUserspaceYaml   = "../../../examples/go-tc-counter/kubernetes-deployment/go-tc-counter.yaml"
+	tcCounterBytecodeYaml      = "../../../examples/config/base/go-tc-counter/bytecode.yaml"
```

Comment:
You can point this to a kustomize file rather than raw yaml now 

And then use `clusters.KustomizeDeployForCluster(ctx, env.Cluster(), tcCounterKustomize)` now I think :)

Reply:
Then we're fully consistent for bpfd core and the examples (always using Kustomize)

Reply:
Done